lib/views/widgets/rich_text_keyboard_toolbar.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:rich_text_composer/views/commons/image_paths.dart';
+
+typedef OnTabCallback = void Function();
+const String packageName = 'rich_text_composer';
+
+class RichTextKeyboardToolBar extends StatelessWidget {
+  final bool? isLandScapeMode;
+  final VoidCallback insertImage;
+  final VoidCallback insertAttachment;
+  final VoidCallback appendRickText;
+
+  final ImagePaths _imagePaths = ImagePaths();
+
+  RichTextKeyboardToolBar({
+    super.key,
+    required this.insertImage,
+    required this.insertAttachment,
+    required this.appendRickText,
+    this.isLandScapeMode,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: isLandScapeMode == true ? MainAxisAlignment.spaceEvenly : MainAxisAlignment.start,
+      children: [
+        _buildIcon(
```

Comment:
how about if use want to add more action on it. Fx: Capture, add link, ....

Reply:
@hoangdat  just create a widget and add it at keyBroadToolbar paramter